src/domUpdates.js

```diff
@@ -342,33 +490,80 @@ function filteredRecipeCards(recipeInput) {
   }
 }
 
-function getRandomUser() {
-  const index = getRandomInt(usersData.length);
-  const randomUser = usersData[index];
+function getRandomUser(users) {
+  const index = getRandomInt(users.length);
+  const randomUser = users[index];
   return randomUser;
 }
+// function getRandomUser() {
+//   return fetchUsers().then(users => {
+//     const index = getRandomInt(users.length);
+//     const randomUser = users[index];
+//     return randomUser;
+//   });
+// }
+
+
+// function displayUserRecipes(users) {
+//   const user = users.find(user => user.name === userName);
+
+//   if (user) {
+//     savedRecipePage.innerHTML = '';
+
+//     user.recipesToCook.forEach(recipe => {
+
+//           const cardHTML = `
+//             <div class="featured-recipe-box" id="${recipe.id}">
+//               <img class="card-image" src="${recipe.image}" alt="${recipe.name}">
+//               <h2 class="card-title">${recipe.name}</h2>
+//               </div>
+//               <div class=remove-recipe>
+//               <button class="remove-button">REMOVE</button>
+//             </div>
+//           `;
+//           savedRecipePage.innerHTML += cardHTML;
+
+//       })
+//     }
+
+// }
+
+function displayUserRecipes(usersData) {
+  // Check if usersData is an array
+  if (!Array.isArray(usersData)) {
```

Comment:
Having these extra conditionals feels like a lot of extra work.  I might set it up so that in the future, any time `displayUserRecipes` is called, you only pass in an array as an argument.